fil_actor_interface/Cargo.toml

```
@@ -3,23 +3,23 @@ name = "fil_actor_interface"
 description = "Backward compatibility layer for all versions of builtin actor released v8 onwards."
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/ChainSafe/fil-actor-states"
-version = "7.0.0"
+version = "7.0.1"
```

The changes to `fil_actor_interface` are not backward compatible.
```suggestion
version = "8.0.0"
```

---

corrected